pkg/agent/toolbox/git/types.go

```diff
@@ -43,3 +43,8 @@ type GitRepoRequest struct {
 	Username *string `json:"username,omitempty" validate:"optional"`
 	Password *string `json:"password,omitempty" validate:"optional"`
 } // @name GitRepoRequest
+
+type GitCheckoutRequest struct {
+	Path   string `json:"path" validate:"required"`
+	Branch string `json:"branch" validate:"required"`
```

Comment:
I suggest these props to be consistent with `GitCloneRequest ` and to be more explicit:
```golang
	Branch   *string `json:"branch,omitempty" validate:"optional"`
	CommitID *string `json:"commit_id,omitempty" validate:"optional"`
```

P.S. It actually might be better to refactor `CommitID` to `SHA` so that is consistent with our `GitRepository` domain object.